### PR TITLE
dev: Add dark mode styles for article pagination

### PR DESCRIPTION
**Pull request by jcbcapps**
# SC-1025

## Proposed changes
Adds dark mode support for the Pagination component.

---

## Reviewer notes
Since we're waiting for the `Pagination` component to be [merged into ReactUSWDS](https://github.com/trussworks/react-uswds/pull/2188), I thought I should keep the styles separate and NOT create an scss module for the component. With that being said, the only place that the `Pagination` component is currently being used is in the `ArticleList` component, so I added the necessary style updates there. 

Something else to note is that the dark mode toggle in Storybook does not work for `Pagination`. I'm not sure why that is the case, but I was thinking that once the `Pagination` component is merged into ReactUSWDS we'll most likely be removing the component and accompanying storybook stories from this codebase (correct me if I'm wrong in that assumption), so that's why I haven't spent any time to get that working.

In order to test the component, open any file that is using the `ArticleList` component:
- `news.tsx`
- `news-announcements.tsx`
- `orbit-blog.tsx`
- or any other instance you may happen upon

and pass it a `pagination` prop that looks like this:
`<ArticleList
        articles={articles}
       pagination={{ currentPage: 10, totalPages: 24 }}
      />`

Note: passing the `pagination` prop to `ArticleList` is just going to cause the `Pagination` component to render. Actual functionality of the component may vary.

Now navigate to wherever the `Pagination` component will be displayed and toggle between light and dark mode to verify that the styles match what you see in Figma.

## Setup
Make sure to have both the CMS and the portal client running on your local machine. You can do this by
- Running `yarn services:up && yarn dev` in the CMS repo and
- Running `yarn dev` in the portal client with this branch checked out

To toggle between light and dark mode, open the dev tools in your browser when running the portal, then click Application > Local Storage > Set a key/value pair for theme, dark.

---

### As the original developer, I have

- [ ] Met the acceptance criteria, or will meet them in subsequent PRs or stories
  - ... 
- [ ] Created new stories in Storybook if applicable
  - [ ] Checked that all Storybook accessibility checks are passing
- [ ] Created/modified automated unit tests in Jest
  - [ ] Including jest-axe checks when UI changes
- [ ] Created/modified automated E2E tests in Cypress
  - [ ] Including cypress-axe checks when UI changes
  - [ ] Checked that the E2E test build is not failing
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)
- [ ] Requested a design review for user-facing changes
- For any new migrations/schema changes:
  - [ ] Followed guidelines for zero-downtime deploys

### As code reviewer(s), I have

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
  - [ ] Checked that the E2E test build is not failing
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] Considered marking this as accepted even if there are small changes needed
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a designer reviewer, I have

- [ ] Checked in the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a test user, I have

- Run through the [Test Script](hhttps://docs.google.com/spreadsheets/d/1eV8UK0aJZ0qrzjnXf5SJ_uRiV7bpsj3yrhRFEkjOvV4/edit?usp=sharing):
  - [ ] On commercial internet in IE11
  - [ ] On commercial internet in Firefox
  - [ ] On commercial internet in Chrome
  - [ ] On commercial internet in Safari
  - [ ] On NIPR in IE11
  - [ ] On NIPR in Firefox
  - [ ] On NIPR in Chrome
  - [ ] On NIPR in Safari
  - [ ] On a mobile device in Firefox
  - [ ] On a mobile device in Chrome
  - [ ] On a mobile device in Safari
- [ ] Added any anomalous behavior to this PR

---

### CODE REVIEW

File: src/styles/_themes.scss [hunk 1]
```diff
@@ -157,6 +163,12 @@
   --form-input-border: #{color(base-light)};
   --form-input-placeholder: #{$theme-white};
 
+  // Pagination
+  --pagination-page-number-bg: #{$white};
```
Comment (gidjin) [inline src/styles/_themes.scss:167]: Suggestion: I don't see pagination in figma but the white bg on the current pagination seems strange in dark mode. I wonder if it should be the green or something. Confirm with @malworks on what color the bg should be here.

**Review by gidjin — COMMENTED** [review]
Suggestion(non-blocking): Can you add a storybook story to show Article list with pagination?

File: src/components/ArticleList/ArticleList.module.scss [hunk 2]
```diff
@@ -22,3 +22,29 @@
 p.NoArticlesText {
   color: var(--text);
 }
+
```
Comment (abbyoung) [inline src/components/ArticleList/ArticleList.module.scss:25]: Appreciate your comment explaining why this lives here for now + that we don't use pagination anywhere else at the moment. Do you think it's worth adding a comment/TODO here for future us?

File: src/styles/_themes.scss (hunk 1 above)
Comment (jcbcapps) [reply src/styles/_themes.scss:167]: Good catch. Pagination is under "For Eng: Dark Theme 1.0" in Figma down at the very bottom, but I did have the color wrong. Updated!

**Review by gidjin — APPROVED** [review]
LGTM now!

**Pull request merged**